File: Secret_Message/Secret_Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace Sololearn
{
    class Program
    {
  public static string de(string str)
{
string res ="";
    foreach(Char x in str)
    {
     if(Char.IsLetter(x))
 res += Convert.ToChar((122-Char.ToLower(x))+97);
 else res += x;
    }
    return res;
}
        static void Main(string[] args)
        {
      string msg = Console.ReadLine();
      Console.Write(de(msg));
        }
    }
}

File: Secret_Message/Secret_Message.java
import java.util.Scanner;

public class Program
{
public static String de(String str)
{
String res= "";
for(char x : str.toCharArray())
{
if(Character.isLetter(x))
res += (char)((122-Character.toLowerCase(x))+97);
else res += x;
}
return res;
}
   public static void main(String[] args) 
{
   Scanner input = new Scanner(System.in);
   String msg = input.nextLine();
   System.out.print(de(msg));
}
}
